internal/nodes/avi_model_l7_hostname_shard.go

```diff
@@ -217,6 +217,10 @@ func (o *AviObjectGraph) ManipulateSniNode(currentSniNodeName, ingName, namespac
 				} else {
 					sniPool = lib.GetSniPoolName(ingName, namespace, hostname, path, svc)
 				}
+				// Pls decprecate when PGs have http caching
+				if lib.GetNoPGForSNI() && isIngr {
+					sniPool = sniPool + "--" + "policy-to-pool"
```

Comment:
Should we add this to a common function or add policy-to-pool to a const ? sniPool name is being generated two times.

Reply:
Added as a constant.